diff --git a/node_modules/@web/test-runner-commands/dist/index.js b/node_modules/@web/test-runner-commands/dist/index.js
index 8c17a88..11b2c93 100644
--- a/node_modules/@web/test-runner-commands/dist/index.js
+++ b/node_modules/@web/test-runner-commands/dist/index.js
@@ -19,4 +19,7 @@ var setViewportPlugin_js_1 = require("./setViewportPlugin.js");
 Object.defineProperty(exports, "setViewportPlugin", { enumerable: true, get: function () { return setViewportPlugin_js_1.setViewportPlugin; } });
 var snapshotPlugin_js_1 = require("./snapshotPlugin.js");
 Object.defineProperty(exports, "snapshotPlugin", { enumerable: true, get: function () { return snapshotPlugin_js_1.snapshotPlugin; } });
+Object.defineProperty(exports, "SnapshotStore", { enumerable: true, get: function () { return snapshotPlugin_js_1.SnapshotStore; } });
+Object.defineProperty(exports, "getSnapshotPath", { enumerable: true, get: function () { return snapshotPlugin_js_1.getSnapshotPath; } });
+Object.defineProperty(exports, "isSaveSnapshotPayload", { enumerable: true, get: function () { return snapshotPlugin_js_1.isSaveSnapshotPayload; } });
 //# sourceMappingURL=index.js.map
diff --git a/node_modules/@web/test-runner-commands/dist/snapshotPlugin.js b/node_modules/@web/test-runner-commands/dist/snapshotPlugin.js
index 042c5bc..ae04320 100644
--- a/node_modules/@web/test-runner-commands/dist/snapshotPlugin.js
+++ b/node_modules/@web/test-runner-commands/dist/snapshotPlugin.js
@@ -33,6 +33,7 @@ function isSaveSnapshotPayload(payload) {
         throw new Error('You must provide a content option');
     return true;
 }
+exports.isSaveSnapshotPayload = isSaveSnapshotPayload;
 function getSnapshotPath(testFile) {
     const testDir = path_1.default.dirname(testFile);
     const testFileName = path_1.default.basename(testFile);
@@ -40,6 +41,7 @@ function getSnapshotPath(testFile) {
     const fileWithoutExt = testFileName.substring(0, testFileName.length - ext.length);
     return path_1.default.join(testDir, '__snapshots__', `${fileWithoutExt}.snap.js`);
 }
+exports.getSnapshotPath = getSnapshotPath;
 class SnapshotStore {
     snapshots = new Map();
     sessionToSnapshotPath = new Map();
@@ -75,9 +77,7 @@ class SnapshotStore {
         const nameStr = JSON.stringify(name);
         const startMarker = `snapshots[${nameStr}]`;
         const endMarker = `/* end snapshot ${name} */\n\n`;
-        const replacement = updatedSnapshot
-            ? `${startMarker} = \n\`${updatedSnapshot}\`;\n${endMarker}`
-            : '';
+        const replacement = `${startMarker} = \n\`${updatedSnapshot}\`;\n${endMarker}`;
         const content = await this.get(testFilePath);
         let updatedContent;
         const startIndex = content.indexOf(startMarker);
@@ -124,6 +124,7 @@ class SnapshotStore {
         }
     }
 }
+exports.SnapshotStore = SnapshotStore;
 function snapshotPlugin(config) {
     const updateSnapshots = config && config.updateSnapshots;
     const snapshots = new SnapshotStore();
diff --git a/node_modules/@web/test-runner-commands/plugins.mjs b/node_modules/@web/test-runner-commands/plugins.mjs
index a91f5b4..14a841a 100644
--- a/node_modules/@web/test-runner-commands/plugins.mjs
+++ b/node_modules/@web/test-runner-commands/plugins.mjs
@@ -11,6 +11,9 @@ const {
   a11ySnapshotPlugin,
   filePlugin,
   snapshotPlugin,
+  SnapshotStore,
+  getSnapshotPath,
+  isSaveSnapshotPayload,
 } = cjsEntrypoint;
 
 export {
@@ -23,4 +26,7 @@ export {
   setUserAgentPlugin,
   setViewportPlugin,
   snapshotPlugin,
+  SnapshotStore,
+  getSnapshotPath,
+  isSaveSnapshotPayload,
 };
